packages/cli/src/commands/setup/deploy/__tests__/netlify.test.js

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+import path from 'path'
+
+const FIXTURE_PATH = path.resolve(
+  __dirname,
+  './__fixtures__/example-netlify-deploy-project'
+)
+
+beforeAll(() => {
+  process.env.RWJS_CWD = FIXTURE_PATH
+})
+
+afterAll(() => {
+  delete process.env.RWJS_CWD
+})
+
+describe('netlify', () => {
+  it('should call the handler without error', async () => {
+    const netlify = require('../providers/netlify')
+    expect(async () => await netlify.handler({ force: true })).not.toThrow()
+  })
+
+  it('should generate a netlify.toml', async () => {
+    const netlify = require('../providers/netlify')
+    await netlify.handler({ force: true })
+    const toml = fs.readFileSync(
```

Comment:
Do you think we could mock the fs.writeFileSync instead of writing out to disk? This results in a lot of performance issues with tests in CI.

Reply:
Good question. I'll check. That would likely also mean that the two toml files (with correct content) would not be needed to check in to the fixture -- which is better.

Reply:
Automocking FS now